azdev/operations/setup.py

```diff
@@ -76,56 +76,60 @@ def _install_cli(cli_path, deps=None):
         whl_list = " ".join(
             [os.path.join(privates_dir, f) for f in os.listdir(privates_dir)]
         )
-        pip_cmd("install -q {}".format(whl_list), "Installing private whl files...")
+        pip_cmd("install {}".format(whl_list), "Installing private whl files...")
 
     # install general requirements
     pip_cmd(
-        "install -q -r {}/requirements.txt".format(cli_path),
+        "install -r {}".format(os.path.join(cli_path, "requirements.txt")),
```

Comment:
Use the better function `os.path.join` in lieu of `/` join to avoid non-standard path separator on Windows:

```
D:\cli\azure-cli/requirements.txt -> D:\cli\azure-cli\requirements.txt
                ^                                    ^
```